Runtime/Scripts/Types/RGBot.cs

```diff
@@ -7,7 +7,7 @@ public class RGBot
     {
         public long id;
         public string name;
-        public string programmingLanguage;
+        public string platform;
```

Comment:
lets leave both fields in this model as I may need to use programming language to filter out 'local' unity bots